wus-vm/requirements.psd1

```diff
@@ -0,0 +1,7 @@
+# This file enables modules to be automatically managed by the Functions service.
+# See https://aka.ms/functionsmanageddependency for additional information.
+#
+@{
+    # For latest supported version, go to 'https://www.powershellgallery.com/packages/Az'. Uncomment the next line and replace the MAJOR_VERSION, e.g., 'Az' = '5.*'
+     # 'Az' = '10.*'

```

Comment:
this should be uncommented if were using a cmdlet from this library in this project